Makefile

```
@@ -646,3 +646,17 @@ shell-check: ## Run ShellCheck on shell scripts.
 	@echo "🔍 Running ShellCheck inside the Nix dev shell..."
 	@DEVENV_ROOT=$(CURDIR) $(NIX_ALLOW_UNFREE) $(NIX_EXEC) develop $(NIX_FLAGS) .# --command $(MAKE) shell-check
+
+##@ Doppler
+
+.PHONY: doppler-sync
+doppler-sync: ## Sync Doppler secrets (dotfiles/prd) to .env file.
+	@echo "🔐 Syncing Doppler secrets to .env..."
+	@doppler secrets download --project dotfiles --config prd --format env --no-file > .env
+	@echo "✅ .env file updated from Doppler (dotfiles/prd)"
+
+.PHONY: doppler-upload
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)